booking-up-for-beauty/jabieren/booking_up_for_beauty.go

```diff
@@ -0,0 +1,39 @@
+package booking
+
+import (
+       "time"
+)
+
+// Schedule returns a time.Time from a string containing a date
+func Schedule(date string) time.Time {
+	layout := "1/02/2006 15:04:05"
+	t, _ := time.Parse(layout, date)
+	return t
+}
+
+// HasPassed returns whether a date has passed
+func HasPassed(date string) bool {
+	layout := "January 2, 2006 15:04:05"
+	t, _ := time.Parse(layout, date)
+	return t.Before(time.Now())
+}
+
+// IsAfternoonAppointment returns whether a time is in the afternoon
+func IsAfternoonAppointment(date string) bool {
+	layout := "Monday, January 2, 2006 15:04:05"
+	t, _ := time.Parse(layout, date)
+	return 12 <= t.Hour() && t.Hour() < 18
+}
+
+// Description returns a formatted string of the appointment time
+func Description(date string) string {
+	layoutIn := "1/2/2006 15:04:05"
+	layoutOut := "You have an appointment on Monday, January 2, 2006, at 15:04."
+	t, _ := time.Parse(layoutIn, date)
+	return t.Format(layoutOut)
```

Comment:
Well done, always try to use what the language gives you. As far I tested, this is the fastest implementation. 500 nanoseconds per operation.